### PR TITLE
Move repository to DAO module

### PR DESCRIPTION
**Pull request by eyouu**

### CODE REVIEW

**Review by Yura09 — APPROVED** [review]

**Review by vorfolomey — APPROVED** [review]

File: core/src/main/java/greencity/controller/HabitController.java [hunk 1]
```diff
@@ -5,11 +5,7 @@
 import greencity.annotations.ValidLanguage;
 import greencity.constant.HttpStatuses;
 import greencity.dto.PageableDto;
-import greencity.dto.habitstatistic.AddHabitStatisticDto;
-import greencity.dto.habitstatistic.HabitDictionaryTranslationsDto;
-import greencity.dto.habitstatistic.HabitItemsAmountStatisticDto;
-import greencity.dto.habitstatistic.HabitStatisticDto;
```
Comment (antonbakalets) [inline core/src/main/java/greencity/controller/HabitController.java:11]: `HabitStatisticDto` should stay. The constructor that depends on entity should be replaced by mapper.

Comment (antonbakalets) [reply core/src/main/java/greencity/controller/HabitController.java:11]: Most probably the same for other DTOs that are used in controllers. Only DTOs that are mapped as query result can go to dao level.

**Review by antonbakalets — CHANGES_REQUESTED** [review]
AppConstant should definitively stay at controller (main application) level.

**Comment by antonbakalets** [comment]
Here is the problem with this task. The only field from `AppConstant` used is `greencity.constant.AppConstant#DATE_FORMAT` But the conversion from String->LocalDateTime should happen at Controller level, preferably it should be Spring responsibility.

- Please try to change the greencity.dto.filter.FilterPlaceDto#time field type to LocalDateTime.

- if that is quite complicated we could hardcode the date format and create a separate ticket to fix it.

**Comment by eyouu** [comment]
> Here is the problem with this task. The only field from `AppConstant` used is `greencity.constant.AppConstant#DATE_FORMAT` But the conversion from `String->LocalDateTime` should happen at Controller level, preferably it should be Spring responsibility.
> 
> * Please try to change the `greencity.dto.filter.FilterPlaceDto#time` field type to `LocalDateTime`.
> * if that is quite complicated we could hardcode the date format and create a separate ticket to fix it.

For out now just hardcoded the format, but I'll create a separate ticket and will do this task (as it was discussed)

**Review by antonbakalets — APPROVED** [review]

**Review by TarasDovganyuk — APPROVED** [review]

**Pull request merged**